src/lib/perceived-latency-instrumentation.js

```diff
@@ -0,0 +1,61 @@
+/* @flow */
+
+type LogLatencyInstrumentationPhaseParams = {|
+    buttonSessionID : string,
+    phase : string
+|};
+
+type InstrumentationPayload = {|
+    comp? : Object,
+    chunk? : Object,
+    query? : Object
+|};
+
+function getNavigationTimeorigin() : number {
```

Comment:
Let's make this camelCase similar to `window.performance.timeOrigin`
```suggestion
function getNavigationTimeOrigin() : number {
```